#!/usr/bin/env python
PACKAGE = "cvt_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# mode_enum = gen.enum([ gen.const("AUTO", int_t, 0, "Automatic mode"), 
# 					   gen.const("MANUAL", int_t, 1, "Manual mode"), 
# 					   gen.const("ONE_PUSH_AUTO", int_t, 2, "One Push Automatic mode"), 
# 					   ],
# 					   "An enum to set feature modes")

gen.add("auto_exposure", bool_t, 0, "Enable Auto Exposure Feature", False )
gen.add("auto_white_balance", bool_t, 0, "Enable Auto White Balance", False )
gen.add("register_frames", bool_t, 0, "Enable image registration", True )
gen.add("sync_frames", bool_t, 0, "Synchornize frame capturing", True )

exit(gen.generate(PACKAGE, "rgbd_camera", "RGBDCamera"))
